src/mars.d

```diff
@@ -821,7 +821,7 @@ Language changes listed by -transition=id:
                 global.params.debugc = true;
             else if (strcmp(p + 1, "-f") == 0)
                 global.params.debugf = true;
-            else if (strcmp(p + 1, "-help") == 0)
+            else if (strcmp(p + 1, "-help") == 0 || strcmp(p + 1, "-h"))
```

Comment:
`== 0`
Also, it needs to be added to the usage.
But since that's most likely to conflict with  #5804, I would wait on this one.